forest/daemon/src/daemon.rs

```diff
@@ -184,20 +184,6 @@ pub(super) async fn start(config: Config, detached: bool) {
     .unwrap();
     chain_store.set_genesis(&genesis.blocks()[0]).unwrap();
 
-    // Terminate if no snapshot is provided or DB isn't recent enough
```

Comment:
Maybe also re-enable the `download` and `import` part of `calibnet sync check` job?

Reply:
I'd rather merge support for NV17 and then re-enable the entire sync check.